FABulous/FABulous.py

```diff
@@ -125,23 +125,30 @@ def main():
 
     setup_global_env_vars(args)
 
-    args.top = os.getenv("FAB_PROJ_DIR").split("/")[-1]
+    projectDir = Path(args.project_dir).absolute()
```

Comment:
Could we stick to the env var? Otherwise, this would brick the possibility to set the project dir via .env files.

Reply:
I have updated, so it take from the .env than fall back to args